libcpu/aarch64/common/cache.h

```diff
@@ -23,9 +23,13 @@ void rt_hw_cpu_dcache_invalidate(void *start_addr, unsigned long size);
 
 static inline void rt_hw_icache_invalidate_all(void)
 {
-    /* wait for any modification complete */
+    /* wait for any former modification to complete */
```

Comment:
suggest to change "former" to "previous". Former means that the person or object was something, but no longer is, like "former girlfriend".

Reply:
Thanks for your advice.
The `former` here is used to distinguish from the `latter`:

```c
code block <= former
barrier
code block <= latter
```

> New Oxford American Dictionary: denoting the first or first mentioned of two people or things

I learned this from some blogs where it's used to denote the paragraph or something like that. But it may be not correct and confused in this scene. So I will just take you advice.

Thanks again.